tutor/src/screens/assignment-builder/mini-editor/editor.js

```diff
@@ -18,6 +17,13 @@ import DraftButton from '../footer/save-as-draft';
 import PlanMixin from '../plan-mixin';
 import ServerErrorHandlers from '../../../components/error-monitoring/handlers';
 import Course from '../../../models/course';
+import NudgeIsAvailableMessage from '../nudge-is-available-message';
+
+const StyledNudgeIsAvailableMessage = styled(NudgeIsAvailableMessage)`
+  font-size: 14px;
+  line-height: 20px;
+  padding-left: 1.5rem;
+`;
```

Comment:
Not even going to ask what is going on here.

Reply:
yeah, that's yucky but is needed to align the message with the editor :(